src/Microsoft.AspNet.Mvc.TagHelpers/Internal/MissingAttributeLoggerStructure.cs

```diff
@@ -24,15 +24,19 @@ public class MissingAttributeLoggerStructure : ILoggerStructure
         /// </summary>
         /// <param name="uniqueId">The unique ID of the HTML element this message applies to.</param>
         /// <param name="missingAttributes">The missing required attributes.</param>
-        public MissingAttributeLoggerStructure(string uniqueId, IEnumerable<string> missingAttributes)
+        /// <param name="extraValues">Extra values to include in the log structure.</param>
```

Comment:
this parameter doesn't exist.  either add it or restore the parameters to a single line.